css-positioning/absolute.css

```diff
@@ -0,0 +1,48 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  display: flex;
+  flex-wrap: wrap;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 225px;
+  top: 15px;
+
+  /* your code here */
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 175px;
+  max-width: 300px;
+  text-shadow: 2px 2px 4px #000;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
+  position: absolute;
+  width: 300px;
+  height: 300px;
```

Comment:
While this would technically work, I want you to be using positioning properties and not `width` and `height`. This would still cause issues if you adjust the image size which would make you adjust multiple rulesets to account for it.